src/h3lib/lib/algos.c

```diff
@@ -335,6 +335,9 @@ H3Error h3NeighborRotations(H3Index origin, Direction dir, int *rotations,
                             H3Index *out) {
     H3Index current = origin;
 
+    if (dir < CENTER_DIGIT || dir >= INVALID_DIGIT) {
```

Comment:
For my info, what does `CENTER_DIGIT` even do? Returns the current index?

Reply:
Yes, that is my expectation